public/project.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
Let's delete this file.  I am going to delete my `manifest.json` on my branch as well.